src/ParticleEmitter.cc

```diff
@@ -487,6 +498,18 @@ void ParticleEmitter::SetTopic(const std::string &_topic)
   this->dataPtr->topic = _topic;
 }
 
+/////////////////////////////////////////////////
+void ParticleEmitter::SetScatterRatio(float _ratio)
+{
+  this->dataPtr->scatterRatio = _ratio;
```

Comment:
```suggestion
  if (_ratio > 0.0f)
    this->dataPtr->scatterRatio = _ratio;
```

Since the scatter ratio must be > 0, should we only set it if the value passed in is > 0? If so, we should also update the documentation to indicate this in `ParticleEmitter.hh`.

Reply:
after thinking about it more,  I removed the requirement on the value needs to be > 0. It's more of an implementation limitation on the ignition end. Since we should keep the spec as general as possible, I think someone could decide to use 0 to simulate 0 scattering for certain sensors.

Reply:
That's a good point. Everything looks good to me, so feel free to merge unless you're waiting for someone else to look over this as well :+1:

Reply:
alright, will merge after CI builds are done